worker/draft.go

```diff
@@ -1157,8 +1157,9 @@ func (n *node) Run() {
 						if span := otrace.FromContext(pctx.Ctx); span != nil {
 							span.Annotate(nil, "Proposal found in CommittedEntries")
 						}
-						if x.WorkerConfig.LudicrousMode {
-							// Assuming that there will be no error while proposing.
+						if x.WorkerConfig.LudicrousMode && len(proposal.Mutations.GetEdges()) > 0 {
```

Comment:
Skipping ludicrous mode for schema mutations seems to be the right approach. If 'Getedges' is the right mechanism for identifying schema changes then it would be good to have a comment to describe our intent here.